setup.py

```diff
@@ -31,6 +31,8 @@
     "h5py>=3.8.0",
     "lxml>=4.9.2",
     "pyyaml",
+    "docker",
```

Comment:
It is generally better to specify dependency versions to avoid any incompatibility due to user's installed version.